web/sites/default/config/webform.webform.report_park_maintenance_issue.yml

```diff
@@ -58,12 +59,7 @@ elements: |-
         visible:
           ':input[name="what_are_you_trying_to_report"]':
             value: 'Tree hazard'
-      '#markup': |-
-        <div class="alert alert--info next-steps">
-        <h2>Next Steps</h2>
-
-        <p>To report a tree issue, please contact us at <a href="tel:311">3-1-1</a> or <a href="311@portlandoregon.gov">311@portlandoregon.gov</a>.</p>
-        </div>
+      '#markup': '<div class="alert alert--info next-steps"><h2>Next Steps</h2><p>To report a tree issue, please contact us at <a href="tel:+15038238733">823-TREE (8733)</a>.</p></div>'
```

Comment:
For accessibility, we should probably include the full standard phone number with area code here, 503-823-8733, then we can put (TREE) outside of the hyperlink.

Reply:
Yeah, good call. I made the change.